README.md

```diff
@@ -102,103 +73,39 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] Kindly remove all the descriptive section on your README. Leaving it there does not make your README file look professional.